HappyAnding/HappyAnding/Views/ShortcutDetailViews/ReadShortcutView/ReadShortcutView.swift

```diff
@@ -49,8 +51,17 @@ struct ReadShortcutView: View {
         .padding(.vertical, 20)
         .background(Color.Background)
         .onAppear() {
-            shortcutsZipViewModel.fetchShortcutDetail(id: shortcutID) { shortcut in
+            shortcutsZipViewModel.fetchShortcutDetail(id: self.data.shortcutID) { shortcut in
                 self.shortcut = shortcut
+                print("hellohello \(self.$shortcut.unwrap()!)")
+            }
+        }
+        .onChange(of: isEdit) { _ in
+            if !isEdit {
+                shortcutsZipViewModel.fetchShortcutDetail(id: self.data.shortcutID) { shortcut in
+                    self.shortcut = shortcut
+                    print(shortcut)
```

Comment:
프린트문은 지워주시면 좋을 것 같습니다!
아직 디버깅용으로 필요하다면 dump()를 사용하는 것도 좋을 것 같아요!
```suggestion
                    dump(shortcut)
```

Reply:
fetch 함수 쪽은 제가 손댄부분이라 따로 손을 안대신것같아요..ㅎㅎ 제 pr에서 지우겠습니당

Reply:
아하 알겠습니당